### PR TITLE
Update flw-scripted-deployment.md

### PR DESCRIPTION
**Pull request by kschmidt1801**
Add a random space

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/flw-scripted-deployment.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ appliesto:
 ---
 # How to provision Teams at scale for Firstline Workers
 
-Do you need to rapidly onboard a large number of users to Microsoft Teams and configure a streamlined experience for them? You can quickly provision identities, provision teams, and assign all relevant policies to control the end user experience by walking through the following instructions.
+Do  you need to rapidly onboard a large number of users to Microsoft Teams and configure a streamlined experience for them? You can quickly provision identities, provision teams, and assign all relevant policies to control the end user experience by walking through the following instructions.
```
Comment (JohanFreelancer9) [inline Teams/flw-scripted-deployment.md:24]:
```suggestion
Do you need to rapidly onboard a large number of users to Microsoft Teams and configure a streamlined experience for them? You can quickly provision identities, provision teams, and assign all relevant policies to control the end-user experience by walking through the following instructions.
```

**Comment by scanum** [comment]
@MicrosoftHeidi PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MicrosoftHeidi** [comment]
I did a major revising of this article today (and there will be another major change in the coming week. Please review to ensure this isn't going to overwrite that work? Thanks.

**Comment by MicrosoftHeidi** [comment]
My previous comment still stands, this cannot be merged to overwrite content changes done to this article--or more changes that are coming this week. If you can confirm there's no content overwrite taking place, this can be merged.

**Comment by SerdarSoysal** [comment]
Closing this PR because the article has completely changed as Heidi mentioned above.

**Pull request closed without merging**